assets/translations/main.json

```diff
@@ -364,7 +366,9 @@
 	"style-panel.opacity": "Opacity",
 	"style-panel.size": "Size",
 	"style-panel.spline": "Spline",
+	"tool-panel.title": "Tools",
 	"tool-panel.more": "More",
+	"navigation-zone.title": "Navigation toolbar",
```

Comment:
<img width="357" alt="Screenshot 2025-04-14 at 11 14 57" src="https://github.com/user-attachments/assets/2edee840-a4ef-4f63-91a2-8baff61a1ab6" />

I don't think we need to include "toolbar" in these

Reply:
nice catch 👍